pvfactors/tests/test_geometry/test_timeseries.py

```diff
@@ -27,6 +27,11 @@ def test_ts_pvrow():
         cut, df_inputs.shaded_length_front,
         df_inputs.shaded_length_back)
 
+    
+    #check segment index
+    assert len(ts_pvrow.front.list_segments) == 3
+    assert [s.index for s in ts_pvrow.front.list_segments] == [0,1,2]
```

Comment:
thanks @tcapelle ! Just a small pep8 issue here but I'll handle it in the new release 👍

Reply:
I not very fond of pep8 =)